psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -2036,8 +2043,9 @@ void DFHelper::transform() {
 
             } else {
                 // total size allowed, in doubles
+                size_t rem_mem = memory_ - (AO_core_ ? naux_ * nbf_ * nbf_ : 0);
```

Comment:
Does `rem_mem` need an alternate value depending on `release_AO_`?

Reply:
Yes, I think so. Changed it to `(AO_core_ && !release_core_AO_before_metric_ ? naux_ * nbf_ * nbf_ : 0)`.